### ISSUE TITLE
Error: Cannot find module 'lodash'

### ISSUE THREAD

**Issue by aduggin** [lucalanca/grunt-a11y#11]
I get the following error when I try to run grunt-a11y:

```
Loading "a11y.js" tasks...ERROR
>> Error: Cannot find module 'lodash'
```

I used the following to install the plugin

```
npm install grunt-a11y --save-dev
```

Which added the following to package.json

```
"grunt-a11y": "^0.1.1",
```

**Comment by brunowego** [comment]
+1

---

@aduggin to fix by now, use `npm install lodash`.

**Comment by lucalanca** [comment]
@aduggin @brunowego this is fixed on [0.1.2](https://github.com/lucalanca/grunt-a11y/releases/tag/0.1.2).

Please update your package.json :+1: I'm sorry for the HUGE delay.

**Comment by brunowego** [comment]
Thanks @lucalanca :+1:
